### PR TITLE
File Manager fixes

### PR DESCRIPTION
**Pull request by xTibor**
This pull request mainly contains rendering fixes for the File Manager.

* I introduced a Column struct to hold information about the column headers.
* The column widths are now initialized based on the width of the column header labels.
* The ".." entries are now accounted for the column width calculations.
* The ".." entries are now not shown at the scheme root folders.
* Added a 4px left margin to the icons.
* Changed the rendering of the up/down arrows. It's easier to render these glyphs using Unicode characters.

TLDR: http://i.imgur.com/AoSIUyZ.png

### CODE REVIEW

**Comment by jackpot51** [comment]
Thanks very much! The old logic was very very magical, and failed in many ways
